test/e2e/app-dir/server-source-maps/server-source-maps.test.ts

```diff
@@ -356,7 +415,25 @@ describe('app-dir - server source maps', () => {
         ).toEqual(3)
       }
     } else {
-      // TODO: test `next start` with `--enable-source-maps`
+      if (isTurbopack) {
+        // Expect the invalid sourcemap warning only once per render.
+        expect(
+          normalizeCliOutput(next.cliOutput).split('Invalid source map.')
+            .length - 1
+        ).toEqual(
+          process.env.NEXT_TEST_CI
+            ? // unclear why this error is not surfaced in CI
+              0
```

Comment:
This is also `0` for me when I run the test locally (in isolation).

Reply:
Thanks for checking. I double check on a clean clone. What command did you use to run the test?

Reply:
It's Node.js 18 vs 20. Adjusting test assertions